src/pages/Schedule.tsx

```diff
@@ -51,18 +58,14 @@ export const Schedule = () => {
     data: baseScheduleData,
     loading: scheduleLoading,
     error: scheduleError,
-  } = useQuery(COURSES, {});
+  } = useQuery(COURSES, { fetchPolicy: "cache-and-network" });
```

Comment:
Does this actually fix the issue? I figured you would have used `network-only`

Reply:
Oh yeah nvm I think this does do what we want :+1:

Reply:
yeah this works fine, I thought doing cache-and-network would help for querying when it doesn't need to refetch, but it looks like it does the full network query everytime anyways lmao